packages/jest-cli/src/test_scheduler.js

```diff
@@ -192,7 +192,7 @@ export default class TestScheduler {
             onResult,
             onFailure,
             {
-              serial: runInBand,
+              serial: runInBand || testRunners[runner].isSerial,
```

Comment:
what about `serial: runInBand || Boolean(testRunners[runner].isSerial)` to ensure we always pass a boolean down?

Reply:
sure, that makes sense 👍